libs/vscode/tasks/src/lib/nx-task-commands.ts

```diff
@@ -341,15 +354,58 @@ async function promptForList() {
 
 async function promptForMigrate() {
   const telemetry = getTelemetry();
-  telemetry.featureUsed('affected-cli');
-  const task = NxTask.create(
-    {
-      command: 'migrate',
-      flags: [],
-    },
-    cliTaskProvider.getWorkspacePath()
+  telemetry.featureUsed('migrate command');
+
+  const { workspacePath } = await getNxWorkspace();
+
+  const packageJson = await readAndParseJson(
+    join(workspacePath, 'package.json')
   );
-  tasks.executeTask(task);
+
+  if (
+    packageJson.dependencies == undefined ||
+    packageJson.devDependencies == undefined
+  ) {
+    return;
+  }
+
+  const dependencyToMigrate = await new Promise<string | undefined>(
```

Comment:
Couldn't this be a simple `showQuickPick` call? I don't see a reason to manually construct a quick pick here

Reply:
Just tested it and it works. Thanks for the tip. Will go with it.